lib/session.js

```diff
@@ -316,12 +327,48 @@ const selectCurrentSession = async function(account) {
   }
 };
 
+const createEconomischeActorByClaims = async function(claims) {
+  const now = new Date();
+
+  const orgUuid = claims.vo_id;
+  const orgUri = `http://data.lblod.info/id/organisaties/${orgUuid}`;
```

Comment:
Don't use the `vo_id` as part of your URI. Generate a random UUID and attach the `vo_id` as a property if needed.